courseware/intro_data_science.md

```diff
@@ -0,0 +1,87 @@
+
+Each assignment notebook should ideally be adapted to suit the needs of your curriculum and serve the needs of a complete and coherent course. We intend for them to serve as examples (templates) for you to customize extensively. We encourage course instructors and teaching assistants to use different datasets (and variables) for each iteration of their course. Luckily, Data Commons makes this easy.
+
+All material is provided publicly and free of charge, under a Creative Commons License ([CC BY](https://creativecommons.org/licenses/by/4.0/)). If you end up finding any of this material useful and would like to be notified of updates, do drop us a line at courses@datacommons.org.
+
+
+## Why use this?
+
+These materials were designed to:
+
+
+
+* **_Use Real Data_**. Via the Data Commons API, students engage with _real world_ data from the get go – no more stale, synthetic datasets. 
+
+* **_Be Interactive._** Each concept is illustrated with examples that instructors and teaching assistants can tinker with minimal effort, allowing students to learn in a hands-on way.
+
+* **_Easy to adapt_.** The notebook format and Data Commons Python API makes everything modular and easy to edit.
+
+
+## Who is this for?
+
+Teachers, Professors, Instructors, Teaching Assistants, and anyone else developing and teaching data science curriculum. We also believe early practitioners can benefit greatly from the exercises.
+
+As an example, MIT's large [Introduction to Machine Learning](https://introml.odl.mit.edu/) course has adapted several of the examples covered in these notebooks to suit their pedagogical needs. From using the same datasets to dive deeper into the material, to modifying the data/variables to illustrate a similar effect, the adaptations span a wide spectrum. 
+
+
+## How can these be used?
+
+We strongly encourage you to change and adapt these notebooks to fit your needs! You can download any notebook in either .ipynb or .py format by clicking on its link and File > Download.
+
+Datasets can be changed by editing the list of variables queried (see the [“Data Commons for Data Science”](https://colab.research.google.com/drive/1FUQGfoIFr9vwa1wRTmVXVdTYHwr7ntHM) tutorial for more on this); editing framing and questions is as easy as editing text cells.
+
+Some ideas:
+* Add additional cells for any additional topics you want covered.
+* For students with stronger programming skills, ask to implement the methods covered on their own.
+* Some of the questions posed to students in the notebooks are open ended – these can be adapted to discussion sessions with students.
+
+
+## Python Notebooks
+
+* [**Data Commons for Data Science Tutorial**](https://colab.research.google.com/drive/1FUQGfoIFr9vwa1wRTmVXVdTYHwr7ntHM#sandboxMode=true)
+    - A quick tutorial introducing the key concepts of working with the Data Commons Python API. Great for familiarizing yourself with how to adapt datasets to your particular needs.  
+<br />
```

Comment:
were these breaks necessary? what happens if they are removed

Reply:
Removing the sub-bullets fixed the formatting issues I tried to use the <br /> tags to solve. These breaks have been removed.

I ended up using the more markdown-flavorful \\ to align descriptions to links. (See new commit.)